bots/task/__init__.py

```diff
@@ -357,11 +358,22 @@ def find_our_fork(user):
                 return full["full_name"]
     raise RuntimeError("%s doesn't have a fork of %s" % (user, api.repo))
 
-def branch(context, message, pathspec=".", issue=None, **kwargs):
-    current = time.strftime('%Y%m%d-%H%M%M')
+def push_branch(user, branch, force=False):
+    fork_repo = find_our_fork(user)
```

Comment:
Hmm, this is like this because I wanted to be able to call `push_branch` from any bot. Surely you can call the last commit with `cont=False` which would push it all. In my mind I had use-case when you don't know how many commits you are gonna make (such as every new language that  got pulled is new commit). In that case you could just do as many commits as wanted, all with `cont=True` and then call `push_branch()`. But surely it would be possible to call `find_our_fork` in the bot itself and then pass the output into the `push_branch` function.  But your approach would save one call to `find_our_fork` in the majority of bots runs.

Reply:
We push to a branch now in `pull()` as well. So in that case it would need to call `find_our_fork` itself. I think I am more in favor of keeping it as is now.